packages/dashboard-v2/src/components/forms/ResetPasswordForm.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from "prop-types";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
+
+import { TextField } from "../Form/TextField";
+import { Button } from "../Button";
+
+import accountsService from "../../services/accountsService";
+
+const resetPasswordSchema = Yup.object().shape({
+  password: Yup.string().required("Password is required").min(6, "Password has to be at least 6 characters long"),
```

Comment:
can't immediately find where, but I remember reviewing a  schema where password had no requirements, in any case this schema is probably duplicated in a couple of places so it would be best to deduplicate this and define the password requirements in a single place